web_modules/layouts/Page/index.js

```diff
@@ -0,0 +1,70 @@
+import React, { PropTypes } from "react"
+import Helmet from "react-helmet"
+
+const Page = ({ head, body, __url }, context) => {
```

Comment:
ahah ok. I am currently considering using less destructuring as I find it hide some code relation but I will fix that ok.